client/app/scripts/superdesk-packaging/views/add-to-package.html

```diff
@@ -0,0 +1 @@
+<div sd-add-package-dropdown></div>
```

Comment:
Thanks @petrjasek , good to know that, will keep in mind in the future when creating new directives.
(FWIW, this PR can of course stay as is)